docs/content/test_runner/Metrics.md

```diff
@@ -0,0 +1,31 @@
+# Test runner metrics
+
+Metrics available at:
```

Comment:
Maybe not in this PR but here must be a part about enabling and configuring metrics.

Reply:
i agree